sources/packages/backend/libs/src-sql/disbursement-overawards/Bulk-insert-bcsl-disbursement-overaward.sql

```diff
@@ -0,0 +1,30 @@
+/*
+ * Insert BCSL overawards from SFAS integration to disbursement overawards table where 
+ * the overard is different than 0. 
+ */
+INSERT INTO sims.disbursement_overawards(
+		student_id,
+		disbursement_value_code,
+		overaward_value,
+		origin_type,
+		creator) 
+SELECT	sfas_individuals.student_id, 
+	   	E'BCSL', 

```

Comment:
Can you please elaborate on why we need the `E` on lines 12 and 14?

Reply:
Sorry. No need to include the "E" here. I was having an issue and the "E" worked. It's used to escape in Postgres.